CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
-## [TBD] - TBD
+## [1.0.1] - TBD
```

Comment:
Let's go with a 1.1.0 release since we made quite a big change to HiveRunner and that's the biggest dependency this project has.